src/components/widgets/FormWidget/FormWidget.tsx

```diff
@@ -51,7 +53,7 @@ export const FormWidget: FunctionComponent<FormWidgetProps> = (props) => {
                         const field = flattenWidgetFields.find(item => item.key === col.fieldKey)
                         const error = (props.missingFields && props.missingFields[field.key])
                             || props.metaErrors && props.metaErrors[field.key]
-                        const fieldLabel = field.type === 'checkbox' ? null : field.label
+                        const fieldLabel = field.type === 'checkbox' ? null : getFieldTitle(field.label, props.data)
```

Comment:
I think we should use a different approach here.  
Instead of introducing dependencies to Form widget, Table widget and all other widgets, which complicates their interface and will create future problems performance-wise, why not introduce a separate component:
```tsx
<TemplatedTitle widgetId={widgetId} title={field.title} container?={CustomControl} />
// ...
const TemplatedWidget = (props) => {
    const wrapper = props.continer && <props.container title={props.templatedTitle} />
    return wrapper || props.templatedTitle
}
// ...
function mapStateToProps(store, ownProps) {
    const widget = store.view.widgets.find(item => item.id === ownProps.widgetId)
    const bc = store.screens.bo.bc[widget.bcName]
    const dataItem = store.data[widget.bcName].find(item => item.id === bc.cursor)
    return {
        templatedTitle: getFieldTitle(ownProps.title, dataItem)
    }
}
export default connect(mapStateToProps)(TemplatedField)
```

Reply:
Fixed. Separate component created